reference/3.0/Microsoft.PowerShell.Core/About/about_Types.ps1xml.md

```diff
@@ -263,310 +278,327 @@ VersionInfo       : File:             C:\WINDOWS\system32\WindowsPow...
+For example, the `VersionInfo` property of files (`System.IO.FileInfo`
+objects) is a script property that results from using the `FullName`
+property of the `GetVersionInfo` static method of
+`System.Diagnostics.FileVersionInfo` objects.
 
+```xml
 <Type>
-<Name>System.IO.FileInfo</Name>
-<Members>
-<ScriptProperty>
-<Name>VersionInfo</Name>
-<GetScriptBlock>
-[System.Diagnostics.FileVersionInfo]::GetVersionInfo($this.FullName)
-</GetScriptBlock>
-</ScriptProperty>
-</Members>
+  <Name>System.IO.FileInfo</Name>
+  <Members>
+    <ScriptProperty>
+      <Name>VersionInfo</Name>
+      <GetScriptBlock>
+        [System.Diagnostics.FileVersionInfo]::GetVersionInfo($this.FullName)
+      </GetScriptBlock>
+    </ScriptProperty>
+  </Members>
 </Type>
+```
 
 For more information, see the Windows PowerShell Software Development
-Kit (SDK) in the MSDN (Microsoft Developer Network )library
-at http://go.microsoft.com/fwlink/?LinkId=144538.
+Kit (SDK) in the MSDN (Microsoft Developer Network) library
+at <http://go.microsoft.com/fwlink/?LinkId=144538>.
 
-Update-TypeData
+## Update-TypeData
 
 To load your Types.ps1xml files into a Windows PowerShell session, run
-the Update-TypeData cmdlet. If you want the types in your file to take
+the `Update-TypeData` cmdlet. If you want the types in your file to take
 precedence over types in the built-in Types.ps1xml file, add the
-PrependData parameter of Update-TypeData. Update-TypeData affects only
+`PrependData` parameter of `Update-TypeData`. `Update-TypeData` affects only
 the current session. To make the change to all future sessions, export
-the session, or add the Update-TypeData command to your Windows
+the session, or add the `Update-TypeData` command to your Windows
 PowerShell profile.
 
 Exceptions that occur in properties, or from adding properties to an
-Update-TypeData command, do not report errors to StdErr. This is to
+`Update-TypeData` command, do not report errors to `StdErr`. This is to
 suppress exceptions that would occur in many common types during formatting
 and outputting. If you are getting .NET Framework properties, you can work
 around the suppression of exceptions by using method syntax instead, as
 shown in the following example:
 
+```powershell
 "hello".get_Length()
+```
 
 Note that method syntax can only be used with .NET Framework properties.
-Properties that are added by running the Update-TypeData cmdlet cannot
+Properties that are added by running the `Update-TypeData` cmdlet cannot
 use method syntax.
 
-Signing a Types.ps1xml File
+## Signing a Types.ps1xml File
 
 To protect users of your Types.ps1xml file, you can sign the file using
-a digital signature. For more information, see about_Signing.
+a digital signature. For more information, see `about_Signing`.
 
 # SEE ALSO
 
 [about_Signing](about_Signing.md)
 
-Copy-Item (http://go.microsoft.com/fwlink/?LinkID=113292)
+Copy-Item (<http://go.microsoft.com/fwlink/?LinkID=113292>)
```

Comment:
Like this: [Copy-Item](http://go.microsoft.com/fwlink/?LinkID=113292) (`[Copy-Item](http://go.microsoft.com/fwlink/?LinkID=113292)`)?

Also, is [this](https://gist.github.com/nzubair/d7b106994cb85a66909e06a509020855#hyperlinks) still the best guide for links? I just want to make sure the `go.microsoft.com` links and relative path links are fine as they are. (i.e. nothing needs to be done to make sure they also work on MSDN or elsewhere)